python/ray/autoscaler/aws/development-example.yaml

```diff
@@ -57,36 +66,32 @@ file_mounts: {
 }
 
 # List of shell commands to run to initialize the head node.
-head_init_commands:
+setup_commands:
     # Install basics.
     - sudo apt-get update
     - sudo apt-get install -y cmake pkg-config build-essential autoconf curl libtool unzip python
+    # TODO(ekl): are these commands idempotent?
```

Comment:
they are definitely not idempotent

Reply:
I added some || trues which should fix it for now.